src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -42,9 +43,9 @@ def _raise_error_if_not_drawing_classifier_input_sframe(
     if len(dataset) == 0:
         raise _ToolkitError("Input Dataset is empty!")
 
-def create(input_dataset, target, feature=None, 
+def create(input_dataset, target, feature=None, validation_set='auto',
             pretrained_model_url=None, batch_size=256, 
-            num_epochs=100, max_iterations=0, verbose=True):
+            num_epochs=100, max_iterations=0, seed=None, verbose=True):
```

Comment:
I'm not sure there is a lot of value to having a `seed` parameter if it's just being used for the train/validation set split. If the user wants reproducibility they can construct the validation set themselves before calling create. If you do want a `seed` parameter, it should be mentioned in the docstring.

Reply:
Let's just be consistent with our other toolkits for now

Reply:
If consistency with other toolkits is our priority, then I don't think we should have a `seed` parameter here. Other toolkits take a `seed` parameter but I believe all of them will produce the same model given the same seed and the same dataset. That isn't true here.

Reply:
If fixing the seed doesn't make model creation deterministic, then I don't think we should expose it, I agree

Reply:
Fixed.